src/app/client/src/app/app.component.ts

```diff
@@ -234,7 +234,15 @@ export class AppComponent implements OnInit, OnDestroy {
           }
         }
       }
-      this.showUserTypePopup = !localStorage.getItem('userType');
+      // TODO: code can be removed in 3.1 release from user-onboarding component as it is handled here.
+      zip(this.tenantService.tenantData$, this.orgDetailsService.orgDetails$).subscribe((res) => {
```

Comment:
Revisit this logic in the next release properly. I can see multiple places subscribe of orgDetailsService & tenantService. We have to keep the workflow proper at one place for loggedIn or non-loggedIn user.